proxies/live/apiproxy/resources/jsc/SetErrorVars.js

```diff
@@ -0,0 +1,210 @@
+// Regex to validate the id is a valid UUID
+const uuidRegex = /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/gi
+const nhsNumberRegex = /^[0-9]{10}$/gi
+const dobRegex = /^[0-9]{4}-[0-9]{2}-[0-9]{2}$/gi
+
+var data = context.getVariable("request.content.data")
+// var requiredParameters = [
+//   "type",
+//   "attributes/routingPlanId",
+//   "attributes/batchReference",
+//   "attributes/recipients",
+//   "attributes/recipients/0/recipientReference",
+//   "attributes/recipients/0/nhsNumber",
+//   "attributes/recipients/0/dateOfBirth",
+//   "attributes/recipients/0/personalisation",
+// ]
+// var requestParameters = context.getVariable("")
+var seenRecipients = []
+var errors = []
+
+var title
+var detail
+var pointer
+
+// $.data
+if (data) {
+  if (data === null || data === undefined) {
+    title = "Property cannot be null"
+    detail = "The property at the specified location is required, but a null value was passed in the request."
+    errors.push(createErrorObject(title, detail, "/data"))
+  } else {
+
+    // $.data.type
+    pointer = "/data/type"
+    if (data.type) {
```

Comment:
For existence of a property use `typeof data.type === 'undefined'` - this is our check for if the property is undefined, `data.type === null` for our null check and finally we can check for value validity.